DependencyInjection/Configuration.php

```diff
@@ -51,6 +51,12 @@ public function getConfigTreeBuilder()
                                     ->append($this->createConsumersNodeDefinition())
                                 ->end()
                             ->end()
+                            ->arrayNode('old_rabbitmq')
```

Comment:
Because it's oldsound/rabbitmq-bundle.
But I can change it to `old_sound_rabbitmq` or `rabbitmq`if you prefer.